src/SourceBuild/content/eng/tools/BinaryToolKit/CompareBinariesAgainstBaselines.cs

```diff
@@ -9,45 +9,41 @@ public static class CompareBinariesAgainstBaselines
 {
     public static List<string> Execute(
         IEnumerable<string> detectedBinaries,
-        string? allowedBinariesFile,
-        string? disallowedSbBinariesFile,
+        string? baselineFile,
         string outputReportDirectory,
         string targetDirectory,
         Modes mode)
     {
-        Log.LogInformation("Comparing detected binaries to baseline(s).");
+        Log.LogInformation($"Comparing detected binaries to baseline {Path.GetFileName(baselineFile)}.");
 
-        var binariesToRemove = GetUnmatchedBinaries(
+        var unmatchedBinaries = GetUnmatchedBinaries(
             detectedBinaries,
-            allowedBinariesFile,
+            baselineFile,
             outputReportDirectory,
             targetDirectory,
             mode).ToList();
 
         if (mode.HasFlag(Modes.Validate))
```

Comment:
I agree that it's a bit odd. It made more sense to have the validation mode implemented there when "all" was a mode option. I'll move the main chunk of this logic (detecting unmatched binaries) into DetectBinaries.cs and then move the actual validation and removal logic into BinaryTools.cs since they're very small methods.